ci/github-script/get-teams.js

```diff
@@ -0,0 +1,78 @@
+const excludeTeams = [
+  /^voters.*$/,
+  /^nixpkgs-maintainers$/,
+  /^nixpkgs-committers$/,
+]
+
+module.exports = async ({ github, context, core, outFile }) => {
+  const withRateLimit = require('./withRateLimit.js')
+  const { writeFileSync } = require('node:fs')
+  const result = {}
+  await withRateLimit({ github, core }, async (stats) => {
+    /// Turn an Array of users into an Object, mapping user.login -> user.id
+    function makeUserSet(users) {
+      return (
+        users
+          .sort((user1, user2) => (user1.login > user2.login ? 1 : -1))
+          // biome-ignore lint/performance/noAccumulatingSpread: We don't care about time complexity
+          .reduce((acc, user) => ({ ...acc, [user.login]: user.id }), {})
```

Comment:
To ameliorate this, you can use mutation:

```suggestion
.reduce((acc, user) => ({ acc[user.login]: user.id; return acc; }, {})
```

unformatted, but you get the idea.